### PR TITLE
add email settings

### PR DESCRIPTION
**Pull request by ring-ring-ring**
as requested in https://github.com/minvws/nl-kat-coordination/issues/1937

### CODE REVIEW

File: docs/source/technical_design/environment_settings/rocky.md [hunk 1]
```diff
@@ -8,3 +8,8 @@ The following places contain more information about which settings Rocky support
 - [.env-dist](https://github.com/minvws/nl-kat-coordination/blob/main/.env-dist)
 - [Django docs about settings](https://docs.djangoproject.com/en/4.2/topics/settings/)
 - [Django complete settings reference](https://docs.djangoproject.com/en/4.2/ref/settings/)
+
+
+## Email forwarding
```
Comment (ammar92) [inline docs/source/technical_design/environment_settings/rocky.md:13]:
```suggestion
## Email Forwarding
```

File: docs/source/technical_design/environment_settings/rocky.md [hunk 2]
```diff
@@ -8,3 +8,8 @@ The following places contain more information about which settings Rocky support
...
+
+[rocky/rocky/settings.py](https://github.com/minvws/nl-kat-coordination/blob/main/rocky/rocky/settings.py#L102-L122) allows you to set the email settings for your OpenKAT install. Errors will be written to Rockys email logs as specified in the ENV file.
```
Comment (ammar92) [inline docs/source/technical_design/environment_settings/rocky.md:15]:
```suggestion
In [rocky/rocky/settings.py](https://github.com/minvws/nl-kat-coordination/blob/main/rocky/rocky/settings.py#L102-L122) you have the ability to define email settings for your OpenKAT installation. Most settings are by default resolved from the environment variables. Any errors will be written to Rocky's email logs as configured in the settings file.
```

**Review by ammar92 — COMMENTED** [review]
Some suggestions, looks good otherwise

**Review by ammar92 — APPROVED** [review]

File: docs/source/technical_design/environment_settings/rocky.md [hunk 3]
```diff
@@ -8,3 +8,8 @@ The following places contain more information about which settings Rocky support
 - [.env-dist](https://github.com/minvws/nl-kat-coordination/blob/main/.env-dist)
 - [Django docs about settings](https://docs.djangoproject.com/en/4.2/topics/settings/)
 - [Django complete settings reference](https://docs.djangoproject.com/en/4.2/ref/settings/)
+
+
+## Email Forwarding
+
+In [rocky/rocky/settings.py](https://github.com/minvws/nl-kat-coordination/blob/main/rocky/rocky/settings.py#L102-L122) you have the ability to define email settings for your OpenKAT installation. Most settings are by default resolved from the environment variables. Any errors will be written to Rocky's email logs as configured in the settings file.
```
Comment (dekkers) [inline docs/source/technical_design/environment_settings/rocky.md:15]:
```suggestion
## Email Settings

Using the environment variables as defined in [rocky/rocky/settings.py](https://github.com/minvws/nl-kat-coordination/blob/main/rocky/rocky/settings.py#L102-L122) you have the ability to define email settings for your OpenKAT installation. These are the Django email settings, see the [django documentation](https://docs.djangoproject.com/en/4.2/topics/email/#smtp-backend) for more information.
```

**Review by dekkers — APPROVED** [review]

**Pull request merged**